openfisca_us/tests/policy/baseline/hhs/ccdf/income_to_smi_ratio.yaml

```
@@ -0,0 +1,35 @@
+- name: Household size 4, State NY
+  period: 2022
+  input:
+    spm_unit_income: 85_000
+    spm_unit_size: 4
+    state_code: NY
+  output:
+    ccdf_income_to_smi_ratio: 85_000/104_972
```

nit but could you do this for the others too
```suggestion
    ccdf_income_to_smi_ratio: 85_000 / 104_972
```